app/assets/stylesheets/darkswarm/images.css.scss

```diff
@@ -17,6 +14,12 @@
       display: none;
     }
   }
+
+  @media only screen and (max-width: 1024px) {
+    margin-top: 0;
+    margin-bottom: 1em;
```

Comment:
I think we should find consensus on whether we use `rem` or `em` in our designs, don't you think?

Reply:
well, it's a design decision isnt it? do you want the element to scale with the font size or not. I think in most cases you do, so REM should be used virtually everywhere.

I had a look at this one, and I just removed this margin, it's not necessary.